test/runtests.jl

```diff
@@ -515,5 +526,35 @@ using Impute:
         end
     end
 
+    @testset "KNN" begin
+        # Test a case with few variable
+        # (e.g., only a few variables, only )
+        @testset "Data - few variables" begin
+            data = Matrix(dataset("Ecdat", "Electricity"))
+            X = add_missings(data)
+
+            knn_imputed = impute(copy(X), Impute.KNN(; num_nn=3, dist=Euclidean(), context=Context(; limit = 1.0)))
+            mean_imputed = impute(copy(X), Fill(; context=Context(; limit=1.0)))
+
+            # If we don't have enough variables then SVD imputation will probably perform
```

Comment:
Might want to update the comments to not reference SVD during KNN imputation ;)

Reply:
I removed.